build/musicbrainz/scripts/docker-entrypoint.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
-set -e
+set -e -o pipefail
 
 # liblocal-lib-perl < 2.000019 generates commands using unset variable
 eval "$(perl -Mlocal::lib)"
 
 set -u
 
+declare -p | grep -P 'MUSICBRAINZ|NODE|PATH|PERL|POSTGRES' > /container-runtime.env
```

Comment:
You are sure that we don't want all ENVs? HOME is one that should probably be forwarded and probably some are missing we don't know about yet.

Reply:
`HOME` is defined the same for cron jobs, but we want neither `BASH_ENV` nor all variables defined by the shell itself (`PWD`, `SHLVL`, `TERM`…) except maybe `HOSTNAME` which is not defined otherwise.

I just [changed the entry point](https://github.com/metabrainz/musicbrainz-docker/pull/169/commits/147980f281b5b416def96c6a8644b75c78d64bbb#diff-7720845733b990d5e10ba1f99eb9f89cR14-R21) to exclude known unwanted variables rather than to allow known variables only, so that it is possible to define custom variables from Docker/Compose. All of already declared variables can also be redefined from Docker/Compose but `PATH`.